shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,58 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 4096
+    nic_hotplug:
+        modprobe_module =
+    block_hotplug:
+        modprobe_module =
+    unattended_install, svirt_install:
+        kernel = images/${os_variant}-${vm_arch_name}/vmlinuz
+        initrd = images/${os_variant}-${vm_arch_name}/initrd.img
+        unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+            cdrom_unattended = images/${os_variant}-${vm_arch_name}/ks.iso
+            syslog_server_proto = udp
+        # ARCH dependent things
+        aarch64:
+            grub_file = /boot/grub/grub.conf
+            install_timeout = 7200
+        ppc64:
+            no guest_s3, guest_s4
+            mem_chk_cmd = numactl --hardware | awk -F: '/size/ {print $2}'
+            netdev_peer_re = "(.*?): .*?\\\s(.*?):"
+            unattended_install..floppy_ks:
```

Comment:
The "unattended_install..floppy_ks" is filtered-out in RHEL.7 on ppc64/ppc64le. I'm not 100% certain on that but isn't floppy supported only on x86 in qemu? Also note that on Fedora we have the floppy_ks disabled on all archs as floppy ks is not really supported, maybe it's time to do the same here...

Reply:
Note if that wasn't clear, I meant to ask you to filter-out floppy at least for ppc64/ppc64le and remove those custom settings. But ideally filter-out floppy everywhere. Should someone need it, they can enable it again...

Reply:
... I mean, have you tested that variant? Does it work?

Reply:
Floppy devices won't be supported in RHEL.8, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#removed_device_drivers

Reply:
@luckyh thanks for info